gef.py

```diff
@@ -8848,6 +8848,60 @@ def do_invoke(self, argv: List[str]) -> None:
             err("No address mapping information found")
             return
 
+        # Parse args:
+        #  - `-a` / `--addr`:
+        #    - filter by address -> parses the next arg as an int or asks gdb
+        #    the value
+        #  - `-n` / `--name`:
+        #    - filter based on section name
+        #  - If nothing is specified, print a warning and guess the type
+        current_type = 0 # 0:None, 1:Address, 2:Name
+
+        addrs = {}
+        names = []
+
+        for arg in argv:
+            if arg in ("-a", "--addr"):
+                current_type = 1
+                continue
+            if arg in ("-n", "--name"):
+                current_type = 2
+                continue
+
+            if current_type == 1: # Address
+                if self.is_integer(arg):
+                    addr = int(arg, 0)
+                else:
+                    addr = safe_parse_and_eval(arg)
+
+                if addr is None:
+                    warn(f"Failed to parse `{arg}` as an address")
```

Comment:
This f-string is not necessary since `arg` will always have teh same value here

Reply:
arg could be many things here, it's addr that is None.

Reply:
`arg` is not None, it is just something that gdb couldn't interpret

Reply:
Fair, read too fast